stubs/enterReplies.js

```diff
@@ -1 +1,27 @@
-module.exports = ['Hi', 'Target Acquired', 'Firing', 'Hello friend.', 'Gotcha', 'I see you']
\ No newline at end of file
+introduction = 'Please join hackbot in welcoming @user to the Rogue Hack Lab!!! :tada:';
+
+icebreakers = [
+  'What\'s your favorite 1990\'s cartoon?',
+  'What did you have for breakfast?',
+  'What\'s one skill you would like to develop?',
+  'What\'s your favorite letter of the alphabet and why?',
+  'How old were you when you first used a computer?',
+  'What\'s your favorite spot for a beer?',
```

Comment:
Let’s change this to be a little more age inclusive as we have some <21 people join.

Reply:
Definitely, that's a good point!